src/main/webapp/app/tasks/task-list/task-list.component.ts

```diff
@@ -1,28 +1,84 @@
-import { ChangeDetectionStrategy, Component, EventEmitter, Inject, Input, Output } from '@angular/core';
+import { CdkDragDrop, moveItemInArray } from "@angular/cdk/drag-drop";
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Inject,
+  Input,
+  Output,
+  ViewEncapsulation,
+} from "@angular/core";
+import { MatSnackBar } from "@angular/material/snack-bar";
+import { forkJoin } from "rxjs";
 
-import { Task } from '../task';
-import { TaskService } from '../task.service';
+import { Task } from "../task";
+import { TaskService } from "../task.service";
+import { sortTasksByStatus } from "../utils";
 
 /**
  * A list of tiny tasks.
  */
 @Component({
-  selector: 'tiny-task-list',
-  templateUrl: './task-list.component.html',
-  styleUrls: ['./task-list.component.scss'],
-  changeDetection: ChangeDetectionStrategy.OnPush
+  selector: "tiny-task-list",
+  templateUrl: "./task-list.component.html",
+  styleUrls: ["./task-list.component.scss"],
+  encapsulation: ViewEncapsulation.None,
```

Comment:
It is because dragged item loses its style. Because the dragged item is attached to the body, instead of the component. 
Maybe a better solution can be found when further investigated.

Reply:
ViewEncapsulation.None? 
Because the dragged element is attached to the body and loses its style that is specified in the component.